src/build.sh

```diff
@@ -97,6 +100,13 @@ function kernel_build()
     llvm='LLVM=1 '
   fi
 
+  if [[ -n "$clean" ]]; then
+    build_clean "$output_kbuild_flag"
+
+    ret="$?"
+    return "$ret"
```

Comment:
You don't need to save `$?` in the `ret` variable, just return `$?` directly.

Reply:
You are right, I already made the required changes. :)